index.html

```diff
@@ -1431,6 +1593,44 @@ <h3 id="lifetime-AudioContext" class="informative">
             be able to observe garbage collections.
           </p>
         </section>
+        <section>
+          <h3>
+            System resources associated with an AudioContext
+          </h3>
+          <p>
+            <a>AudioContext</a>s should be considered expensive objects.
+            Creating an <a>AudioContext</a> often involves creating a
+            high-priority thread, and using a low-latency system audio stream,
+            both having an impact on energy consumption. Creating more than one
+            <a>AudioContext</a> in a document is most of the time unnecessary.
+          </p>
+          <p>
+            Additionally, a user-agent can have an implementation-defined
+            maximum number of <a>AudioContext</a>, after which any attempt to
+            create a new <a>AudioContext</a> will fail, <span class=
+            "synchronous">throwing <code>NotSupportedError</code></span>.
+          </p>
+          <p>
+            <a href="#widl-AudioContext-suspend-Promise-void">suspend</a> and
+            <a href="#widl-AudioContext-resume-Promise-void">close</a> allow
+            authors to <dfn id="releasing">release system resources</dfn>.
+            <a href="#releasing">Releasing system resources</a> means releasing
+            the system resources such as threads, processes, audio streams, but
```

Comment:
Is this required?  Chrome only releases threads and streams when closing.  Suspending just pauses the audio stream.